modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -6,34 +6,71 @@ endif::[]
+. Select *Salesforce CRM Content User*.
+. *Save*.
+
+== API Community Manager Entitlement
+
+Installation requires Anypoint API Community Manager entitlement for your Anypoint organization.
 
-.. Select *Salesforce CRM Content User*.
-.. Click *Save*.
-. The Anypoint API Community Manager entitlement for your Anypoint organization.
-+
 To check the entitlement:
 
-.. Log in to your Anypoint Platform organization.
-.. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
+. Log in to your Anypoint Platform organization.
+. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using the EU cloud, navigate to `+https://eu1.anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using MuleSoft Government Cloud, navigate to `+https://gov.anypoint.mulesoft.com/accounts/api/profile+`.
-.. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
-. Identity provider data.
+. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
+
+== A Domain with Enabled Communities
+
+To set up and register a domain:
+
+. Log in to your API Community Manager https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
+. Go to *Salesforce Set-up* by selecting the gear icon and selecting *Set-up*.
+. Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
+. Enter a domain name for your organization.
++
+The name can include up to 34 letters, numbers, and hyphens.
+. Select *Check Availability*.
++
+If the name is already taken, choose a different one.
+. Select *Register Domain*.
+
+You will receive a notification email when your domain is registered and your
+subdomain name is ready for testing. This process can take a few minutes.
+You can continue with the next steps while you wait for the email.
+
+NOTE: After you complete API Community Manager installation and configuration, you can configure a full custom domain
+name, also known as a vanity domain. When you're ready, refer to the Salesforce knowledge article
+https://help.salesforce.com/articleView?id=000336819&type=1&mode=1[Run your Salesforce Community under a custom domain, role=external, window=blank].
+
+Using the domain name you selected in the previous step, follow the steps in
+https://help.salesforce.com/articleView?id=networks_enable.htm&type=5[Enable Salesforce Communities, role=external, window=blank]
+to enable communities inside your Salesforce organization.
+
+Wait to receive the notification email that your domain is registered, and then deploy your domain:
+
+. Log in to your API Community Manager https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
+. Go to *Salesforce Set-up* by clicking the gear icon and selecting *Set-up*.
+. Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
```

Comment:
Repeats steps above...so is a separate task???
See previous edits to these steps in Lines 44-46.